corehq/form_processor/tests/test_case_dbaccessor.py

```diff
@@ -563,6 +564,29 @@ def test_case_has_transactions_since_sync(self):
             CaseAccessorSQL.case_has_transactions_since_sync(case1.case_id, "foo", datetime.utcnow())
         )
 
+    def test_get_all_cases_modified_since(self):
+        case1 = _create_case(user_id="user1")
+        case2 = _create_case(user_id="user1")
+        middle = datetime.utcnow()
+        time.sleep(.01)
```

Comment:
nit: ~might be better to actually set `received_on` so that it's clear that that's the property we're testing rather than sleeping